### DISCUSSION TITLE
My code gets stucked when a unicode element is found

### DISCUSSION THREAD

**Discussion by Adrij1993** [dask/dask#9860] [Q&A]
The block of code arguments where problem starts: My excel file gets read, previous lines get executed but the following block stops.
for j in range(len(sd)):
        if sd.iloc[j,19]<7.500:
           count = count
        else:
           count = count + 1   
           avg_shf = (sd.iloc[j,23] + sd.iloc[j,24])*0.5

Error: 
[2880 rows x 29 columns]
(0, numpy.datetime64('2022-09-01T05:30:00.000000000+0530'))
(1, numpy.datetime64('2022-09-02T05:30:00.000000000+0530'))
(2, numpy.datetime64('2022-09-03T05:30:00.000000000+0530'))
(3, numpy.datetime64('2022-09-04T05:30:00.000000000+0530'))
(4, numpy.datetime64('2022-09-05T05:30:00.000000000+0530'))
(5, numpy.datetime64('2022-09-06T05:30:00.000000000+0530'))
(6, numpy.datetime64('2022-09-07T05:30:00.000000000+0530'))
(7, numpy.datetime64('2022-09-08T05:30:00.000000000+0530'))
(8, numpy.datetime64('2022-09-09T05:30:00.000000000+0530'))
(9, numpy.datetime64('2022-09-10T05:30:00.000000000+0530'))
(10, numpy.datetime64('2022-09-11T05:30:00.000000000+0530'))
(11, numpy.datetime64('2022-09-12T05:30:00.000000000+0530'))
(12, numpy.datetime64('2022-09-13T05:30:00.000000000+0530'))
(13, numpy.datetime64('2022-09-14T05:30:00.000000000+0530'))
(14, numpy.datetime64('2022-09-15T05:30:00.000000000+0530'))
(15, numpy.datetime64('2022-09-16T05:30:00.000000000+0530'))
(16, numpy.datetime64('2022-09-17T05:30:00.000000000+0530'))
(17, numpy.datetime64('2022-09-18T05:30:00.000000000+0530'))
(18, numpy.datetime64('2022-09-19T05:30:00.000000000+0530'))

Traceback (most recent call last):
  File "C:\Users\anushriya\Desktop\Adrij\demo.py", line 31, in <module>
    avg_shf = (sd.iloc[j,23] + sd.iloc[j,24])*0.5
TypeError: coercing to Unicode: need string or buffer, numpy.float64 found
>>> j
47
>>> i
18
>>> sd.iloc[j,23]
u'E4R'
>>> type(sd.iloc[j,23])
<type 'unicode'>

**Comment by ncclementi** [comment]
hi @Adrij1993 , Dask now has a [Discourse forum](https://dask.discourse.group/t/introduce-yourself/22/23) for usage-questions. When you post there it will be helpful if you can provide more context, to help folks understand better your issue.

We'll see you there! :)

**Reply by Adrij1993** [reply]
Thanks @ncclementi

**Comment by Adrij1993** [comment]
Thanks for your reply.
I am checking.

Regards.
